.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@jamesrweb @yevdyko
```

Comment:
Minor: missing EOL

Reply:
True, this repo doesn't have linting or formatting like the `react` one does though, maybe we can create another repo with shared configs for eslint, prettier, etc and use the shared configs in this repo and in the `react` one and any future ones also. WDYT?

My main focus right now is working on the Q5 option, which is in-progress already on my side, and also the migration to P5 version 2.x.x but I haven't started that one. Would you have time to take that on now? If not, I will get to it after the Q5 work I am doing.

Reply:
Yeah, I can take care of it this weekend. Thanks for bringing it up!

Reply:
@jamesrweb Could you please create a `workflows` repository under the `P5-wrapper` organization, as I don't have the permissions to do that?

Reply:
Thanks, I have joined the organization and am going to update the `workflows` repository tomorrow